freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -322,3 +314,39 @@ def image_has_auto_rebuild_tag(self, image):
             if set(auto_rebuild_tags) & set(tags):
                 return True
         return False
+
+    def _create_original_to_rebuilt_nvrs_map(self, event):
+        """
+        Creates mapping of original build NVRs to rebuilt NVRs in advisory.
+        Including NVRs of the builds from the blocking advisories
+
+        :param BotasErrataShippedEvent event: Freshmaker's event that is
```

Comment:
Could you just use `self.event` instead?

Reply:
Yes, you are right